sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClient.java

```diff
@@ -973,9 +973,15 @@ public boolean getSubscriptionExists(String topicName, String subscriptionName)
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Response<Boolean> getSubscriptionExistsWithResponse(String topicName, String subscriptionName,
                                                                Context context) {
-        final Response<SubscriptionProperties> subscriptionWithResponse =
-            getSubscriptionInternal(topicName, subscriptionName, context);
-        return getEntityExistsWithResponse(subscriptionWithResponse);
+        try {
+            final Response<SubscriptionProperties> subscriptionWithResponse =
+                getSubscriptionInternal(topicName, subscriptionName, context);
+            return getEntityExistsWithResponse(subscriptionWithResponse);
+        } catch (ServiceBusManagementErrorException | ResourceNotFoundException ex) {
+            HttpResponse response = ex.getResponse();
+            return new SimpleResponse<>(response.getRequest(), response.getStatusCode(),
+                response.getHeaders(), false);
+        }
```

Comment:
I wonder 

1. If we can make error behavior the same as the async client?
2. If we can have the same error behavior for all entity types (queue, topic, subscription)?

By updating - "getEntityExistsWithResponse" to

```java
try {
    final boolean exists = getEntityOperation.getValue() != null;
    return new SimpleResponse<>(getEntityOperation.getRequest(), 
       getEntityOperation.getStatusCode(),
       getEntityOperation.getHeaders(), exists);
} catch(RuntimeException error) {
   final RuntimeException mappedError = ServiceBusAdministrationAsyncClient.mapException(error);
   if (mappedError instanceof ResourceNotFoundException) {
     ResourceNotFoundException notFoundError = (ResourceNotFoundException) mappedError;
     final HttpResponse response = notFoundError.getResponse();
     return new SimpleResponse<>    
       (response.getRequest(), response.getStatusCode(),
       response.getHeaders(), false);
   } else {
       throw error;
   }
}
```

we could make `mapException` in the async client [here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java#L2462-L2490) package private and use above.

What do you think?

If so we may need to also adjust the changelog as it fixes "isExists" api for all 3 entity types.